include/wx/dynarray.h

```diff
@@ -24,8 +24,12 @@
     template <typename T>
     inline void wxVectorShrinkToFit(std::vector<T>& v)
     {
+    #if __cplusplus >= 201103L
```

Comment:
Maybe add a check for MSVC as well, it is still at `199711L`.
E.g. as in `sting.h`:
```c++
#if (__cplusplus >= 201103L || wxCHECK_VISUALC_VERSION(10))
```

Reply:
Yes, thanks for noticing this.